tests/integration/_cases/monitors/monitors-run-help.trycmd

```diff
@@ -15,28 +15,28 @@ Options:
       --header <KEY:VALUE>
           Custom headers that should be attached to all requests
           in key:value format.
-      --auth-token <AUTH_TOKEN>
-          Use the given Sentry auth token.
   -s, --schedule <schedule>
           Configure the cron monitor with the given schedule (crontab format). Enclose the schedule
           in quotes to ensure your command line environment parses the argument correctly.
+      --auth-token <AUTH_TOKEN>
+          Use the given Sentry auth token.
```

Comment:
We should not be including the `--auth-token` in the output for `sentry-cli monitors run --help` if auth token authentication is no longer supported

Reply:
I am going to add a commit to tweak this